src/test/java/com/okexchain/msg/TestGov.java

```diff
@@ -1,86 +1,88 @@
-
-
-        MsgContractBlockedListProposal msg = new MsgContractBlockedListProposal();
-        msg.init(new PrivateKey("75dee45fc7b2dd69ec22dc6a825a2d982aee4ca2edd42c53ced0912173c4a788".toUpperCase()));
-
-        String[] contractAddresses = new String[]{"okexchain1hw4r48aww06ldrfeuq2v438ujnl6alsz0685a0","okexchain1qj5c07sm6jetjz8f509qtrxgh4psxkv32x0qas"};
-
-        Message messages = msg.produceContractDeploymentWhitelistProposal(
-                "String title",
-                "String description",
-                contractAddresses,
-                true,
-                "100.000000000000000000"
-        );
-
-        JSONObject res = msg.submit(messages, "0.03", "2000000", "");
-        System.out.println(res.toJSONString());
-        try {
-            boolean succeed = msg.isTxSucceed(res);
-            System.out.println("tx " + (succeed ? "succeed": "failed"));
-        } catch (Exception e) {
-            System.out.println(e.toString());
-        }
-    }
-
-    @Test
-    public void testContractBlockedListProposalValue() throws IOException {
-        EnvBase env = EnvInstance.getEnv();
-        env.setChainID("okexchainevm-8");
-        EnvInstance.getEnv().setRestServerUrl("http://localhost:8545");
-        env.setDenom("okt");
-
-
-        MsgContractDeploymentWhitelistProposal msg = new MsgContractDeploymentWhitelistProposal();
-        msg.init(new PrivateKey("75dee45fc7b2dd69ec22dc6a825a2d982aee4ca2edd42c53ced0912173c4a788".toUpperCase()));
-
-        String[] distributorAddresses = new String[]{"okexchain1hw4r48aww06ldrfeuq2v438ujnl6alsz0685a0","okexchain1qj5c07sm6jetjz8f509qtrxgh4psxkv32x0qas"};
-
-        Message messages = msg.produceContractDeploymentWhitelistProposal(
-                "String title",
-                "String description",
-                distributorAddresses,
-                true,
-                "100.000000000000000000"
-        );
-
-        JSONObject res = msg.submit(messages, "0.03", "2000000", "");
-        System.out.println(res.toJSONString());
-        try {
-            boolean succeed = msg.isTxSucceed(res);
-            System.out.println("tx " + (succeed ? "succeed": "failed"));
-        } catch (Exception e) {
-            System.out.println(e.toString());
-        }
-    }
-
-}
+//package com.okexchain.msg;
```

Comment:
It’s best to delete unused code directly

Reply:
removed